permissions/plugin-splunk-devops-extend.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "splunk-devops-extend"
+paths:
+- "com/splunk/splunkins/splunk-devops-extend"
+developers:
+- "fengxx"
+- "araza_splunk"
```

Comment:
`araza_splunk` does not existing in Artifactory (see this repo's README) so will prevent these permissions to be applied for everyone.